generators/app/constants.js

```diff
@@ -78,15 +71,17 @@ module.exports.TEMPLATE_FILES = [
   UTILS_PATH,
   `${UTILS_PATH}/colors.js`,
   `${UTILS_PATH}/array.js`,
+  `${SCREENS_PATH}/Dashboard/screens/Home/assets`,
+  `${SCREENS_PATH}/Dashboard/screens/Home/assets/logo.svg`,
   `${SCREENS_PATH}/Dashboard/screens/Home/index.js`,
+  `${SCREENS_PATH}/Dashboard/screens/Home/styles.scss`,
   `${SCREENS_PATH}/Dashboard/index.js`,
   `${COMPONENTS_PATH}/Routes/components/AuthenticatedRoute.js`,
   `${COMPONENTS_PATH}/Routes/constants.js`,
-  `${COMPONENTS_PATH}/Routes/index.js`,
+  `${COMPONENTS_PATH}/Routes/styles.js`,
```

Comment:
Why leaving the styles file?

Reply:
This file was already in the bootstrap, do not modify it. I think that this file shouldn't exist but it doesn't enter into the scope of this pr

Reply:
However the diff shows that you deleted the file, so you can remove the styles file also

Reply:
@FrankIglesias `Routes/index.js` appears below 😇

Reply:
Exactly! Check `TEMPLATE_FILES` const below